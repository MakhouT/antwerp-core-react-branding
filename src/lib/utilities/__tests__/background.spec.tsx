import * as React from "react";

import * as A from "../..";
import { renderAndMatchSnapshot } from "../../../__tests__/__fixtures__/testUtils";

describe("Background", () => {
    it("Renders black", () => {
        // Arrange
        const element = <A.Background
            type={A.BackgroundStyle.Black}
        />;

        // Act and assert
        renderAndMatchSnapshot(element);
    });

    it("Renders danger", () => {
        // Arrange
        const element = <A.Background
            type={A.BackgroundStyle.Danger}
        />;

        // Act and assert
        renderAndMatchSnapshot(element);
    });

});
