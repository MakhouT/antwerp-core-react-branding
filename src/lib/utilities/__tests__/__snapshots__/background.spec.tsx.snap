// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Background Renders black 1`] = `
<div
  className="u-bg-black u-bg-white-text"
  style={undefined}
/>
`;

exports[`Background Renders danger 1`] = `
<div
  className="u-bg-danger u-bg-white-text"
  style={undefined}
/>
`;
